opennms-config-model/src/main/java/org/opennms/netmgt/config/eventd/.castor.cdr

```diff
@@ -0,0 +1,2 @@
+#Tue Feb 28 16:56:15 EST 2017
```

Comment:
Ah yeah, forgot to convert to `XmlTestNoCastor` and remove that stuff after it passed. (I've been making sure the code still works with castor before removing support.)